test/common/testlib.py

```diff
@@ -1792,15 +1792,16 @@ def check_journal_messages(self, machine=None):
             self.allowed_messages.append("Failed to connect to coredump service: No such file or directory")
             self.allowed_messages.append("Failed to connect to coredump service: Connection refused")
 
+        #
         # HACK: pybridge bugs
-        if os.environ.get("TEST_SCENARIO") == "pybridge":
```

Comment:
I liked the greppability of this, but as you wish...

Reply:
Me too -- but you'll see why we have to do this in my next PR, and in #18757